src/Components/WebView/Platforms/Wpf/src/BlazorWebView.cs

```diff
@@ -0,0 +1,50 @@
+using System;
+using System.Windows;
+using System.Windows.Controls;
+using Microsoft.Web.WebView2.Wpf;
+
+namespace Microsoft.AspNetCore.Components.WebView.Wpf
+{
+    public sealed class BlazorWebView : Control, IDisposable
```

Comment:
I'll add a commit to apply my changes from https://github.com/dotnet/MobileBlazorBindings/pull/333 to here.

Reply:
Excellent - thanks!